client/sections.js

```diff
@@ -461,6 +461,12 @@ const sections = [
 			},
 		],
 	},
+	{
+		name: 'jetpack-search',
+		paths: [ '/jetpack-search' ],
```

Comment:
Scan and Backup use paths not prefixed with `jetpack-`, but we need `/search` for other things.

Reply:
makes sense.